### PR TITLE
Logtest

### PR DESCRIPTION
**Pull request by anniephan**
WIP

### CODE REVIEW

File: log/log_test.go [hunk 1]
```diff
@@ -1 +1,160 @@
...
+	log.SetOutput(buf)
+	outputf(LevelDebug, string1, nil)
+
+	// test 1: outputf correctly prints string
+	if !strings.Contains(buf.String(), string1) {
+		t.Fail()
+	}
+
+	(*bytes.Buffer).Reset(buf)
+	outputf(LevelDebug-1, string1, nil)
```
Comment (lziest) [inline log/log_test.go]: can you use the actual Level constant?

File: log/log_test.go [hunk 2]
```diff
@@ -1 +1,160 @@
...
+
+	// test 1: outputf correctly prints string
+	if !strings.Contains(buf.String(), levelPrefix[LevelInfo]) {
+		t.Fail()
+	}
+	return
+}
+
+func TestDebugf(t *testing.T) {
+	const string1 = "asdf123"
```
Comment (lziest) [inline log/log_test.go]: can we use an actual format string? :)

File: log/log_test.go [hunk 3]
```diff
@@ -1 +1,160 @@
...
+)
+
+func TestOutputf(t *testing.T) {
+	const string1 = "asdf123"
+	buf := new(bytes.Buffer)
+
+	log.SetOutput(buf)
+	outputf(LevelDebug, string1, nil)
+
+	// test 1: outputf correctly prints string
```
Comment (lziest) [inline log/log_test.go]: if you planning to add more test, please do. If not, remove "test 1" here and at other places.

**Comment by grittygrease** [comment]
Please address the comments above and squash into one commit.

File: log/log_test.go (hunk 1 above)
Comment (anniephan) [inline log/log_test.go]: It turns out that there is no Level constant lower than LevelDebug. So it seems like there is no need to include that second test... Thanks! I will fix.

File: log/log_test.go [hunk 4]
```diff
@@ -1 +1,134 @@
 package log
+
+import (
+	"bytes"
+	"log"
+	"strings"
+	"testing"
+)
+
+const string1 = "asdf123"
```
Comment (grittygrease) [inline log/log_test.go]: Can you update this name to not have "1" in it?

Comment (anniephan) [inline log/log_test.go]: Ok

**Pull request closed without merging**